docs/src/main/asciidoc/feature-management.adoc

```diff
@@ -0,0 +1,279 @@
+
+When an endpoint is blocked because the feature it specifies is disabled, `IDisabledFeaturesHandler` will be invoked. By default, an HTTP 404 is returned. This can be overridden by implementing `IDisabledFeaturesHandler`.
+
+[source,java,indent=0]
+----
+@Component
+public class DisabledFeaturesHandler implements IDisabledFeaturesHandler{
+
+    @Override
+    public HttpServletResponse handleDisabledFeatures(HttpServletRequest request, HttpServletResponse response) {
+        ...
+        return response;
+    }
+
+}
+----
+
+===== Routing
+
+Certain routes may expose application capabilities that are gated by features. These routes can be redirected if a feature is disabled to another endpoint.
+
+[source,java,indent=0]
+----
+@GetMapping("/featureT")
+@FeatureGate(feature = "feature-t" fallback= "/oldEndpoint")
+@ResponseBody
+public String featureT() {
+    ...
+}
+
+@GetMapping("/oldEndpoint")
+@ResponseBody
+public String oldEndpoint() {
+    ...
+}
+----
+
+=== Built-In Feature Filters
+
+There are a few feature filters that come with the `azure-spring-cloud-feature-management` package. These feature filters are not added automatically, but can be setup in an `@Configuration` for use.
+
+==== AlwaysOnFilter
+
+This filter always returns true.
+
+==== PercentageFilter
+
+This filter provides the capability to enable a feature based on a set percentage.
+
+[source,yaml,indent=0]
+----
+feature-management:
+  feature-v:
+    enabled-for:
+      -
+        name: PercentageFilter
+        parameters:
+          percentage-filter-setting: 50
+----
+
+==== TimeWindowFilter
+
+This filter provides the capability to enable a feature based on a time window. If only `time-window-filter-setting-end` is specified, the feature will be considered on until that time. If only start is specified, the feature will be considered on at all points after that time. If both are specified the feature will be considered valid between the two times.
+
+[source,yaml,indent=0]
+----
+feature-management:
+  feature-v:
+    enabled-for:
+      -
+       name: TimeWindowFilter
+        parameters:
+          time-window-filter-setting-start: "Wed, 01 May 2019 13:59:59 GMT",
+          time-window-filter-setting-end: "Mon, 01 July 2019 00:00:00 GMT"
+----
+
+==== TargetingFilter
+
+This filter provides the capability to enable a feature for a target audience. An in-depth explanation of targeting is explained in the targeting section below. The filter parameters include an audience object that describes users, groups, and a default percentage of the user base that should have access to the feature. Each group object that is listed in the target audience must also specify what percentage of the group's members should have access. If a user is specified in the users section directly, or if the user is in the included percentage of any of the group rollouts, or if the user falls into the default rollout percentage, then that user will have the feature enabled.
```

Comment:
In the part where you mention the targeting section below, could you link to it to shortcut the experience? 

Also for the third sentence, maybe this makes it clearer:
"For each group object that is listed in the target audience, a percentage is required which defines the percentage of that group's members which has access to the feature."